libs/pages/elewa/about-us/src/lib/components/history-carousel/history-carousel.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="wrapper">
+    <div class="counter-col">
+        <h2>{{peoplecount}}+</h2>
+        <p class="count-text ">People trained</p>
+    </div>
+    <div class="counter-col">
+        <h2>{{careercount}}k</h2>
+        <p class="count-text ">Careers launched</p>
+    </div>
+    <div class="counter-col">
+        <h2>{{projectcount}}+</h2>
+        <p class="count-text ">Projects finished</p>
+    </div>
+    <div class="counter-col">
+        <h2>{{investorcount}} </h2>
+        <p class="count-text ">Investors</p>
+    </div>
+</div>
```

Comment:
This is not a carousel, rename component to social-impact-enterprise-section

should contain figures and reusable (text and image) banner as well. With correct text as shown in design